src/documents/views/ReviewDetailView.js

```diff
@@ -59,18 +59,37 @@ const ReviewDetailView = ({
     myProfile && hasPermission(myProfile, 'change_datareview');
   const allowViewReview =
     myProfile && hasPermission(myProfile, 'view_datareview');
+  const allowStartValidation =
+    myProfile && hasPermission(myProfile, 'add_validationrun');
 
   const [downloadFileMutation] = useMutation(FILE_DOWNLOAD_URL);
 
-  const {loading, data, error} = useQuery(DATA_REVIEW, {
-    variables: {
-      id: Buffer.from('DataReviewNode:' + reviewId).toString('base64'),
+  const {loading, data, error, startPolling, stopPolling} = useQuery(
```

Comment:
I noticed a lot of info being requested in the poll queries:

<img width="767" alt="image" src="https://user-images.githubusercontent.com/10283712/117062266-43d41380-acf1-11eb-87df-56b2d2439b8a.png">

Some of the requested content seems unrelated to validation (e.g. study fields, dataReview.creator fields, dataReview.versions) and should probably be removed from the query. 

You may also want to revise the part of the query that fetches the validation runs. Right now the response will keep growing as more validation runs are created bc this query is fetching all of the validation runs on every poll. 

Maybe the poll requests should ask for the latest/last validation run? We might have to modify the validation runs query to make it easy to get the latest one.